controller/Makefile

```diff
@@ -50,6 +50,14 @@ deploy: build dev-release restart
 runserver:
 	python manage.py runserver
 
+postgres:
+	docker start postgres || docker run --restart="always" -d -p 5432:5432 --name postgres postgres:9.4.1
+	docker exec postgres createdb -U postgres deis 2>/dev/null || true
+	@echo "To use postgres for local development:"
+	@echo "    export export PGHOST=192.168.1.97"
```

Comment:
:sob: Sorry about that, I missed the line from development. The IP address should be `127.0.0.1`, which works on linux and should work on OSX.